solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -569,6 +646,31 @@ contract RandomBeacon is Ownable {
         }
     }
 
+    /// @notice Punishes the given operators by banning their sortition pool rewards.
+    /// @dev By the way, this function releases gas deposits made by operators
+    ///      during their registration. See `registerOperator` function. This
+    ///      action makes punishments cheaper gas-wise.
+    /// @param ids IDs of punished operators.
+    /// @param punishmentDuration Duration of the punishment period in seconds.
+    function punishOperators(uint32[] memory ids, uint256 punishmentDuration)
```

Comment:
So maybe just `banFromRewards`?

Reply:
Done in https://github.com/keep-network/keep-core/pull/2707/commits/c97dba585f021ce3e8ec7fb37e2665a4f7577541 as part of #2707